apps/blockchain/test/blockchain/account/repo_test.exs

```diff
@@ -184,6 +184,32 @@ defmodule Blockchain.Account.RepoTest do
 
       assert found_code == code
     end
+
+    test "caches machine code and then always reads it from cache", %{state: state} do
+      address = <<1>>
+      code = <<5>>
+
+      {updated_repo, {:ok, found_code}} =
+        state
+        |> Account.reset_account(address)
```

Comment:
Just for my own understanding, why do we have to reset the account here? Does that do something with the cache?

Reply:
@germsvel it creates blank account in the database